layouts/funding_calls/list.html

```diff
@@ -40,154 +37,80 @@ <h4>Potential funding sources</h4>
 <!-- Code for other funding sources ends here -->
 
 <script>
-// Functions need to be run when the DOM is ready
-document.addEventListener("DOMContentLoaded", function (event) {
-    
-    // Takes a URL and calls the callback function pass the data fetched
-    function getDataFromUrl(url, load_callback) {
-      var req = new XMLHttpRequest();
-      req.overrideMimeType("application/json");
-      req.open("GET", url, true);
-      // The function call for fetching data and display
-      req.onload = function () {
-        load_callback(JSON.parse(req.responseText).items);
-      };
-      req.send(null);
-    };
-
-    // Code for funding sources starts here
-    function addFundingsCalls(dataArray) {
-      var dataList = [];
-      // Not insterested if the deadline have already passed
-      dataArray.forEach((item) => {
-        if (new Date(item.deadline) >= new Date().setHours(0,0,0,0)){
-          dataList.push(item);
+  // Takes a URL and fetchs all open funding calls with label "Data Centre"
+  async function getDCFundingCalls(url, label="Data Centre") {
+    try {
+      // Fetch DC funding directly from Anubis using API
+      const response = await fetch(url);
+      if (!response.ok) {
+        throw new Error(`HTTP error! status: ${response.status}`);
+      }
+      const allData = await response.json();
+      // Check if there are 'calls' in the response and they are a list
+      if (!allData?.calls || !Array.isArray(allData.calls)) {
+        return [];
+      }
+      const allFund = allData.calls.filter((f) => f.labels.includes(label));
```

Comment:
Sorry to skip it before but would like to check this as well like.

```suggestion
      const allFund = allData.calls.filter((f) => f.labels && Array.isArray(f.labels) && f.labels.includes(label));
```

to avoid exception throw if a call object lacks a labels property.

Reply:
Agreed 👍